server/rules.go

```diff
@@ -0,0 +1,316 @@
+package server
+
+import (
+	"encoding/json"
+	"io"
+	"net/http"
+
+	"github.com/rs/zerolog/log"
+
+	"github.com/RedHatInsights/insights-operator-utils/responses"
+	"github.com/RedHatInsights/insights-results-aggregator/storage"
+	"github.com/RedHatInsights/insights-results-aggregator/types"
+)
+
+// likeRule likes the rule for current user
+func (server *HTTPServer) likeRule(writer http.ResponseWriter, request *http.Request) {
```

Comment:
Maybe better to move all this vote stuff, to separate file?

Reply:
separate file for ~40 lines?

Reply:
yes, I think logically it would be better. Like in file `rules` you have operations with rules, in file `votes` you have operations with votes.

Reply:
And it will be ~80 lines.

Reply:
then we would need file for disabling/enabling rules and creating/deleting which is too much I think.

Reply:
well, about `disabling/enabling rules and file creating/deleting` I'm not sure, because for example number functions that are working with votes is 6, I think it's enough for separate file, no?

Reply:
I don't think so, anyway we can move it later in different PR